lib/pluginUtils.js

```diff
@@ -104,7 +104,8 @@ function checkVersion (fn) {
   const requiredVersion = meta.fastify
 
   const sanitizedVersion = this.version.replace(/-rc.\d+$/, '')
```

Comment:
This line should not be needed if we switch to use `includePrerelease`
```suggestion
```

Reply:
Yes. This line should be removed.